src/pages/console/formbuilder/customize.mdx

```diff
@@ -0,0 +1,107 @@
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-add.mp4"/>
+</video>
+
+## Reorder form inputs with drag and drop
+
+Rearrange form inputs vertically or horizontally.
+1. Click and drag a form input
+2. Move form input above, below, left, or right of another form input
+3. Drop when a blue bar appears, indicating the form input's new placement
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-drag-and-drop.mp4"/>
+</video>
+
+## View form as end user
+
+If you want to quickly test your form customizations, select **View as end user** to view the customized form fully functioning within Studio.
+
+**Note:** If your form is connected to data model, then form data will be saved to the cloud upon clicking **Submit**. Careful to not accidentally add unintended records into a production database.
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-view-as-end-user.mp4"/>
+</video>
+
+## Configure form spacings (paddings and gaps)
+
+Add spacing to your form and between inputs. The spacing editor allows you to set the following spacing values (from left-to-right):
+- **Horizontal gap**: the space between horizontally arranged inputs.
+- **Vertical gap**: the space between vertically arranged inputs.
+- **Padding**: the space around the form inputs and the form's border.
+
+Spacing values can either be a CSS length value (`px`, `rem`, `em`, `%`) or a reference to your theme object's spacing value (`xss`, `medium`, `large`).
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-spacing.mp4"/>
+</video>
+
+## Configure options for Select Field or Radio Group Field
+
+[Select Field](https://ui.docs.amplify.aws/react/components/selectfield) and [Radio Group Field](https://ui.docs.amplify.aws/react/components/radiogroupfield) require a set of options for your users to choose from. For example, a "Status" input can only have the options "Not started", "In progress", and "Done".
+
+1. Select a **Select Field** or **Radio Group Field** input
+2. Go to **Data > Options**
+3. Enter available options line-by-line or paste in a JSON array, such as `["Not started", "In progress", "Done"]`
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-options.mp4"/>
+</video>
+
+## Enable list inputs
+
+Every form input can made to accept multiple values. The form input gets "wrapped" around by a "list input" component to allow your users to add, update, and remove items.
+
+![Screenshot of "Accept multiple values" checkbox](/images/console/formbuilder/customize-accept-multiple-values.png)
+
+1. Select a form input in a form that's not connected to a data model
```

Comment:
`a form input in a form` is redundant. Could probably simplify to `an input in a form`